source/includes/administration/_notification_categories.md

```diff
@@ -0,0 +1,235 @@
+        {
+            "language":"fr",
+            "type":"title",
+            "text":"trial-tt"
+        }
+    ]
+}
+```
+Create a notification category.
+
+Required | &nbsp;
+---------- | -----------
+`organizationId`<br/>*UUID* | The organization id of the notification category.
+`createdAt`<br/>*string* | The date on which the notification category was created.
+`updatedAt`<br/>*string* | The date on which the notification category was updated.
+`icon`<br/>*string* | The icon used when displaying the notification category.
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of that the translation is in.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating. (Must be one of: title, body, description, url slug)
+
+<!-------------------- UPDATE NOTIFICATION CATEGORY -------------------->
+
+#### Update notification category
+
+`PUT /content/categories/:id`
+
+```shell
+# Update notification category
+curl -X PUT "https://cloudmc_endpoint/v2/rest/content/categories/:id" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request-body"
+```
+> Request body example:
+
+```json
+{
+    "id": "dc921a2a-16c2-4bc3-8ed6-bcb571bae241",
+    "organizationId":"3e824205-f507-4ffe-9891-a8addf3cf0e9",
+    "icon": "fa fa-envelope",
+    "translations": [
+        {
+            "language": "en",
+            "text": "Test-notif-reseller",
+            "type": "title"
+        },
+        {
+            "language": "fr",
+            "text": "test-notif-reseller",
+            "type": "title"
+        }
+    ]
+}
+```
+Update a notification category.
+
+Required | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification category.
+`organizationId`<br/>*UUID* | The organization id of the notification category.
+`createdAt`<br/>*string* | The date on which the notification category was created.
+`updatedAt`<br/>*string* | The date on which the notification category was updated.
+`icon`<br/>*string* | The icon used when displaying the notification category.
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of that the translation is in.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating. (Must be one of: title, body, description, url slug)
+
+
+<!-------------------- DELETE NOTIFICATION CATEGORY -------------------->
+
+
+### Delete notification category
+
+`DELETE /content/categories/:id`
+
+```shell
+# Delete a notification category
+curl "https://cloudmc_endpoint/v2/content/categories/:id" \
+   -X DELETE -H "MC-Api-Key: your_api_key"
+
+```
+
+Delete a notification category. You must ensure that this category has no associated
```

Comment:
You must ensure that this category has no associated **notification**.

Reply:
40448fe47ff6582d909c3046be19ca1060e34bf7